CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Multiple Swift versions support  
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#8191](https://github.com/CocoaPods/CocoaPods/issues/8191)
+
+* Adds app spec project integration.  
```

Comment:
this was wrongly placed in 1.6.0.beta.2 section. Moved it in a separate commit as part of this PR